wiki/Beatmapping/Flow/en.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - linear flow
+  - oval flow
+  - circular flow
+---
+
+# Flow
+
+**Flow** is a mapping concept that describes natural and intuitive movement patterns of a player during gameplay. Flow, or the lack thereof, is almost always related to the [osu! game mode](/wiki/Game_mode/osu!) due to the two-dimensional playfield. By utilising the vast variety of [patterns](/wiki/Beatmap/Pattern) and slider shapes possible in osu! beatmaps, mappers are able to direct a player's cursor around the playfield in various ways, yielding different effects on the gameplay experience.
```

Comment:
> Flow, or the lack thereof, is almost always related to the [osu! game mode](/wiki/Game_mode/osu!) due to the two-dimensional playfield

This is not true as it also applies to the gamemode osu!catch

Reply:
I left a note for someone (maybe it could be the future you?) to pick up